temporalio/nexus/_operation_context.py

```diff
@@ -146,25 +147,30 @@ def _get_workflow_event_links(
     def _add_outbound_links(
         self, workflow_handle: temporalio.client.WorkflowHandle[Any, Any]
     ):
+        # If links were not sent in StartWorkflowExecutionResponse then construct them.
```

Comment:
Is it valuable to even parse the link? Can we just take any link w/ workflow event on it from the response and set it on outbound?

Reply:
I see. So I wonder then what the server links has that makes it better than the one manually built here, but I can understand if the server one should be the canonical one.